index.js

```diff
@@ -98,6 +99,49 @@ app.post("/github_discord_urls", async (req, res) => {
   }
 });
 
+async function insert_user_details_to_db() {
```

Comment:
problem: email & password should be coming in as function params - don't depend on the global params

Reply:
I didn't get the logic though. What's the reason?